### PR TITLE
Added a "Tables and Matrices" section to the Creating a New Tut…

### PR DESCRIPTION
**Pull request by mtekman**
This can wait until after GCC if it ruins someone's tutorial

### CODE REVIEW

File: topics/contributing/tutorials/create-new-tutorial-content/tutorial.md [hunk 1]
```diff
@@ -574,6 +574,51 @@ Rendered:
 > Something really bad can happen here!
 {: .warning}
 
+
+## Tables and Matrices
```
Comment (bebatut) [inline topics/contributing/tutorials/create-new-tutorial-content/tutorial.md]: I would put this section in the "Content" section. The "Improving the learning experience" is more dedicated to the boxes

Comment (bebatut) [reply topics/contributing/tutorials/create-new-tutorial-content/tutorial.md]: btw, thanks for adding that

**Comment by mtekman** [comment]
(I didn't dismiss anything....)

**Comment by erasche** [comment]
Per @bebatut suggestion IRL moved into proper subsections. LGTM.

**Comment by mtekman** [comment]
looks good!

**Pull request merged**